sdk/formrecognizer/ai-form-recognizer/samples-dev/prebuilt/prebuilt-tax.us.w2.ts

```diff
@@ -73,99 +72,99 @@ export interface TaxUsW2 {
  */
 export interface TaxUsW2Fields {
   /**
-   * `TaxUsW2` "W2FormVariant" field
+   * IRS W2 Form variant. This field can have the one of the following values: 'W-2', 'W-2AS', 'W-2CM', 'W-2GU' or 'W-2VI'
    */
   w2FormVariant?: fr.DocumentStringField;
   /**
-   * `TaxUsW2` "TaxYear" field
+   * Form tax year
    */
   taxYear?: fr.DocumentStringField;
   /**
-   * `TaxUsW2` "W2Copy" field
+   * W2 form copy version along with printed instruction realted to this copy
    */
   w2Copy?: fr.DocumentStringField;
   /**
    * `TaxUsW2` "Employee" field
    */
   employee?: fr.DocumentObjectField<TaxUsW2Employee>;
   /**
-   * `TaxUsW2` "ControlNumber" field
+   * W2 Form control number. IRS W2 form field d
```

Comment:
`field d` is intentional?

Reply:
The control number is literally "Box D" on the W2, so this is intentional, though the message could be better. These comments come directly from the model schemas.